package.json

```diff
@@ -8,7 +8,8 @@
     "empty.js"
   ],
   "scripts": {
-    "test": "mocha --reporter list test/*.js"
+    "test": "mocha --reporter list test/*.js",
+    "postinstall": "node scripts/setup-symlinks.js"
```

Comment:
End users don't need this link to happen so I'd prefer if the module didn't do it for all users.